hbase-server/src/main/java/org/apache/hadoop/hbase/coprocessor/BaseRegionObserver.java

```diff
@@ -180,11 +178,11 @@ public void preCompactSelection(final ObserverContext<RegionCoprocessorEnvironme
 
   @Override
   public void postCompactSelection(final ObserverContext<RegionCoprocessorEnvironment> c,
-      final Store store, final ImmutableList<StoreFile> selected) { }
+      final Store store, final List<StoreFile> selected) { }
```

Comment:
Downstream consequence of a RegionObserver API fix. This method should not receive Guava's ImmutableList as type, it should simply be List. Using ImmutableList to represent the fact that this list is, and should be, immutable, is nice, but binds a LimitedPrivate interface to the vagaries of Guava's frequent breaking changes as it itself evolves.